docs/specifying-return-values.md

```diff
@@ -63,8 +63,8 @@ than 4 parameters, the convenience methods won't work. Use the variant
 that takes an `IFakeObjectCall` instead:
 
 ```charp
-A.CallTo(objectCall => fakeShop.SomeCall(…)
-                               .ReturnsLazily(objectCall => calculateReturnFrom(objectCall));
+A.CallTo(() => fakeShop.SomeCall(…)
+                       .ReturnsLazily(objectCall => calculateReturnFrom(objectCall));

```

Comment:
I think there's still a missing closing parenthesis here at the end of the first line?